packages/smooth_app/lib/pages/crop_page.dart

```diff
@@ -493,6 +507,20 @@ class _CropPageState extends State<CropPage> {
       return false;
     }
   }
+
+  void _showErrorDialog() {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    showDialog<void>(
```

Comment:
`await`

Reply:
Mmm that's meaningless here, as we're not statement after.
Why would it be necessary?

Reply:
I'm not sure if it's "necessary", but it would be cleaner and more consistent with the rest of the `showDialog` calls in the app.

Reply:
I've added a `Future` to the method, which should be better